habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/ValidatePythonCodeMojo.java

```diff
@@ -0,0 +1,80 @@
+package org.technologybrewery.habushu;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+import org.technologybrewery.habushu.exec.PoetryCommandHelper;
+
+/**
+ * Leverages the pylint package to validate both source and test Python
+ * directories using Poetry's run command.
+ */
+@Mojo(name = "validate-python-code", defaultPhase = LifecyclePhase.PROCESS_CLASSES, requiresDependencyResolution = ResolutionScope.COMPILE)
+public class ValidatePythonCodeMojo extends AbstractHabushuMojo {
+
+    protected static final String PYLINT_PACKAGE = "pylint";
```

Comment:
S: s/b "LINT_PACKAGE" so we could, in theory change it more easily.

Reply:
Addressed, thanks Ryan for the suggestion!